website_portal_contract/static/src/js/tour_test_contract.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2017 Laslabs Inc.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_portal_contract.tour_test_contract', function(require) {
+    'use strict';
+
+    var tour = require('web_tour.tour');
+    var base = require('web_editor.base');
+
+    tour.register(
+        'test_contract_view',
+        {
+            url: '/my/home',
+            name: 'Test website portal contract view',
+            test: true,
+            wait_for: base.ready(),
+        },
+        [
+            {
+                content: 'Click on Your Contracts',
+                trigger: "a:contains('Your Contracts')",
+            },
```

Comment:
hrmmm this doesn't make sense. Everything works fine now in my browser when I run the test, but it's not happy in travis.

One of the travis tests are passing currently, and when I pushed the last WIP commit, all the tests passed on the [LasLabs ](https://travis-ci.org/LasLabs/contract/jobs/236089022)side.